SWLOR.Game.Server/Feature/ChatCommandDefinition/DMChatCommand.cs

```diff
@@ -406,10 +406,64 @@ private void SetLocalVariable()
 
                     SendMessageToPC(user, "Local string set: " + variableName + " = " + value);
                 });
+
+            _builder.Create("tptag")
+                .Description("Sets a local tag on a target Teleport Object placeable.")
+                .Permissions(AuthorizationLevel.DM, AuthorizationLevel.Admin)
+                .RequiresTarget()
+                .Validate((user, args) =>
+                {
+                    if (args.Length <= 0)
+                    {
+                        return "Missing arguments. Format should be: /TPTag <VALUE>. Example: /TPTag DUNGEON_ENTRANCE";
+                    }
+
+                    return string.Empty;
+                })
+                .Action((user, target, location, args) =>
+                {
+                    if (GetResRef(target) != "tele_obj")
+                    {
+                        SendMessageToPC(user, "This command can only be used on the Teleport Object placeable.");
+                    }
+                    else
+                    {
+                        SetTag(target, args[0]);
+
+                        SendMessageToPC(user, "Tag set to: " + args[0] + ".");
+                    }
+                });
+
+            _builder.Create("destination")
+                .Description("Sets the DESTINATION localstring to the given value on a target.")
```

Comment:
Maybe make this less technical? Something like "Sets the tag of the destination on a teleport object" or similar?

Reply:
Yeah will reword tonight and update this